apps/plugin-browser/src/pages/popup/popup-contents/action-center/log/inference-requests.tsx

```diff
@@ -114,47 +116,82 @@ const InferenceRequestContainer = ({
         {request.status === "pending" ? (
           <Stack alignItems="center" direction="row">
             {cancellationRequested ? (
+              <Tooltip title="Cancellation pending...">
```

Comment:
Maybe we should rethink the positioning of the tooltips here, it's making it slightly harder to hover from the right icon to the left icon because of the placement:

https://github.com/hashintel/hash/assets/42802102/ac7b979d-fb60-481a-a429-cdfd7b6ae544

Reply:
Moved in https://github.com/hashintel/hash/pull/4168/commits/a16e59eb50dd29f18dff88e9341d060c2fedd945